src/app/modules/webhook/webhook.service.ts

```diff
@@ -253,8 +257,9 @@ export const createInitialWebhookSender =
     return sendWebhook(submission.getWebhookView(), webhookUrl).andThen(
       (webhookResponse) => {
         webhookStatsdClient.increment('sent', 1, 1, {
-          responseCode: webhookResponse.response.status.toString(),
+          responseCode: `${webhookResponse.response.status || null}`,
           webhookType: getWebhookType(webhookUrl),
+          isRetryEnabled: `${isRetryEnabled}`,
```

Comment:
`isRetryEnabled` is not what is important. `isRetryEnabled` is a property of the form, not the current webhook call. The property of the webhook call is whether it is the original call or a retry call.

With that, we can have stats later to know how many of our outbound webhook calls are tries.

We could try tagging the retry attempt number, 1, 2, 3, 4, ... but there's a risk that it becomes unbounded if we change retry behaviour, so that's why I was suggesting tragging just `is_retry:0|1`

Reply:
Replied to this in the first PR - repasting it here!
This DD tracking is in the `createInitialWebhookSender`, so it only counts the initial webhook that's sent, which means we don't have to filter out retries. I have added another tag isRetryEnabled in case we want to know whether retries have been enabled on the different categories of webhooks.

Reply:
Ah understood, thanks for the clarification 👍 

I still don't think there is much value in tracking `isRetryEnabled` on the call metrics though 🤔 . As a property of the form, it would make more sense into the daily stats of the webhook feature.

But regardless, 2 additional tag values aren't much, so it's fine to leave it there for now. 👍